IO/Ingestor/JsonIngestor.cs

```diff
@@ -0,0 +1,73 @@
+﻿using CommunAxiom.Commons.Ingestion.Attributes;
+using CommunAxiom.Commons.Ingestion.Configuration;
+using CommunAxiom.Commons.Ingestion.Validators;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+
+namespace CommunAxiom.Commons.Ingestion.Ingestor
+{
+    [IngestionType(IngestorType.JSON)]
+    public class JsonIngestor : IngestorBase, IIngestor
+    {
+        private IEnumerable<DataSourceConfiguration> _configurations;
+        private readonly IFieldValidatorLookup _fieldValidatorLookup;
+
+        public JsonIngestor(IFieldValidatorLookup fieldValidatorLookup)
+        {
+            _fieldValidatorLookup = fieldValidatorLookup;
+        }
+
+        public void Configure(IEnumerable<DataSourceConfiguration> configurations)
+        {
+            _configurations = configurations;
```

Comment:
This should receive IEnumerable<FieldMetadata>.
This method lacks a visitor pattern to configure FieldMetadata objects in the enumerable based on standard validations associated to their specific FieldType (we aren't sure that the Validators list will be populated properly (actually, accounting for multi version support, we know for a fact that Validators will change over time, some will be deprecated, others created..)